Tests/print_tests.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+from dojo import Dojo
+
+
+class TestPrintCreation(unittest.TestCase):
+
+    """ Test cases to for successful and failed  printing of lists"""
+
+    def test_can_print_names_of_people_in_room(self):
+
+        """ test successful printing of names of people in room_name on screen"""
+
+        self.dojo = Dojo()
+        self.dojo.create_room("office", ["kenya"])
+        self.dojo.add_person('Kevin', 'Oriels', 'Fellow')
+        print_room = self.dojo.print_room("kenya")
+        self.assertEquals(print_room, "Kenya Contains Kevin Oriels")
```

Comment:
The print function always returns `None`. A simple way to do it is to try on your interactive shell, print(print(4)). You will see that the first answer will be 4 and the second one will be None.. `None` is not equal to "Kenya Contains Kevin Oriels". 

To test a print function, you'll need to "trap" the output into a string and compare the output with the expected output.